test/toNumber.js

```diff
@@ -0,0 +1,24 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('ToNumber', function () {
+  context('given a number', function () {
```

Comment:
Maybe, you could create a `context` for all JS types 👀 

[number, boolean, string, ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures)

Reply:
Sounds good!
I added the tests for all JS types